docs/Third-party.md

```diff
@@ -1,281 +1,268 @@
+|   DWI > Connectome   |  ✓   |           |     ✓      |  ✓  |     |        |    ✓    |        |     |
+|  PET > Surface (2)   |      |           |            |  ✓  | ✓⟡  |        |         |   ✓⟡   |  ✓  |
+|     PET > Volume     |      |           |            |     | ✓⟡  |   ✓    |         |   ✓⟡   |  ✓  |
+| Statistics > Surface |      |           |            |     |     |   ✓    |         |        |     |
+</div>
 
-Nonetheless, if you are using an older version of `ANTs`, make sure to have the following environment variables defined:
+*✓⟡ : you only need to install ITK if you plan to perform partial volume correction using PETPVC.*
 
-```bash
-export ANTSPATH="/path/to/your/ANTs/"
-export PATH=${ANTSPATH}:${PATH}
-```
+??? warning "CAT12 toolbox and Clinica < `v0.3.7`"
+    Starting from Clinica `v0.3.7`, the [**CAT12**](http://dbm.neuro.uni-jena.de/cat/) toolbox is no longer needed for the `t1-volume` and `pet-volume` pipelines.
+    For previous versions of Clinica, you will need to download the latest version of the toolbox [here](http://dbm.neuro.uni-jena.de/cat/index.html#DOWNLOAD) and follow the instructions to ensure that your `cat12` folder is located in your `spm/toolbox` folder.
 
-### Convert3D
+Depending on the architecture and OS of your system, setup of third party libraries can change.
+Please refer to each tool’s website for installation instructions :
+___
 
-You can find more details about `Convert3D` [here](http://www.itksnap.org/pmwiki/pmwiki.php?n=Convert3D.Convert3D).
+### ANTs
 
-#### Installation
+To install `ANTs`, download it from [ANTs release list](https://github.com/stnava/ANTs/releases) and follow the instructions on [ANTs wiki](https://github.com/stnava/ANTs/wiki/Compiling-ANTs-on-Linux-and-Mac-OS).
 
-You have two options to install `Convert3D`:
+=== "`ANTs >= 2.5.0` 🔺"
+    We **strongly** recommend installing `ANTs >= 2.5.0` from which **no environment variable are needed**.
 
-- [Use pre-built binaries](http://www.itksnap.org/pmwiki/pmwiki.php?n=Downloads.C3D).
-- [Use the official conda package](https://anaconda.org/conda-forge/convert3d).
+=== "`ANTs < 2.5.0`"
+    Nonetheless, if you are using an older version of `ANTs`, make sure to have the following environment variables defined:
+    
+    ```{ .bash .copy }
+    export ANTSPATH="/path/to/your/ANTs/"
+    export PATH=${ANTSPATH}:${PATH}
+    ```
 
-### Freesurfer
+___
 
-You can find more details about `Freesurfer` [here](http://surfer.nmr.mgh.harvard.edu/).
+### Convert3D
+
+You can find more details about `Convert3D` on their [website](http://www.itksnap.org/pmwiki/pmwiki.php?n=Convert3D.Convert3D). There are
+two options to install it :
+
+- [Use pre-built binaries](http://www.itksnap.org/pmwiki/pmwiki.php?n=Downloads.C3D).
+- [Use the official conda package](https://anaconda.org/conda-forge/convert3d).
 
-#### Installation
+___
 
-##### On Linux
+### FreeSurfer
 
```

Comment:
I realize that the softwares are listed in a totally random order. 
I suggest we use alphabetical order. WDYT ?

Reply:
Yes, they are... I got tricked by the diff 😅 
Sorry for the noise !